src/components/Button.js

```diff
@@ -92,6 +104,8 @@ class Button extends React.PureComponent {
           }
 
           .drui-button--withTextAndIcon {
+            min-width: 124px;
```

Comment:
Like I said, I don't think it's a good idea to set that min-width in this package.
I think it'll be better to set min-width declaration to a custom class when using this component in a separate project.